xclient_gui/desktop/views/register.py

```diff
@@ -62,27 +53,29 @@ def create_password_input(self, layout):
         layout.addWidget(label_password, 1, 0)
         layout.addWidget(self.lineEdit_password, 1, 1)
 
-    def create_login_button(self, layout):
-        button_login = QPushButton('Login')
-        button_login.clicked.connect(self.login)
-        button_login.setStyleSheet("padding-top: 5px; padding-bottom: 5px;")
-        layout.addWidget(button_login, 2, 0, 1, 2)
-        layout.setRowMinimumHeight(2, 75)
+    def create_confirm_password_input(self, layout):
+        label_confirm_password, self.lineEdit_confirm_password = FormInput('Confirm Password:',
+                                                                           "Confirm your password").get_input()
```

Comment:
Add ":" to get "Confirm your password:"

Reply:
Confirm your password is a placeholder. There is no need to add ":" in placeholder.